internal/cmd/root.go

```diff
@@ -7,13 +7,19 @@ import (
 	"github.com/snapp-incubator/nats-blackbox-exporter/internal/logger"
 	"github.com/snapp-incubator/nats-blackbox-exporter/internal/metric"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
 // ExitFailure status code.
 const ExitFailure = 1
 
+var settingsPath string
+
 func Execute() {
-	cfg := config.New()
+	pflag.StringVar(&settingsPath, "settings", "/opt/nats-blackbox-exporter/settings.yml", "Path to settings file")
```

Comment:
Whats your idea about using configPath instead of setting? I think it's more like industry standard